### DISCUSSION TITLE
Audio with tracker popup

### DISCUSSION THREAD

**Discussion by rickschwarz** [f1multiviewer/issue-tracker#191] [Feature requests]
It would be grat to have a feature that makes it possible to select a commentary language and audio device for ourput the audio in the tracker popup window

**Comment by f1multiviewer** [comment]
You mean the telemetry window(s)?

**Reply by rickschwarz** [reply]
I mean in the seperate window of the driver tracker

**Reply by rickschwarz** [reply]
This is seperate from my other request

**Reply by f1multiviewer** [reply]
So you mean the "Driver tracker" stream? If so, you can already do that, either through the player's settings (the settings icon) or by right-clicking and "Set Audio Track". is this what you mean?

**Reply by rickschwarz** [reply]
No, i mean the tracker window that can be opend from you live timing window

**Comment by rickschwarz** [comment]
No, i mean the tracker window that can be opend from you live timing window


Op 3 september 2022 14:23:06 schreef MultiViewer for F1 ***@***.***>:

So you mean the "Driver tracker" stream? If so, you can already do that, either through the player's settings (the settings icon) or by right-clicking and "Set Audio Track". is this what you mean?
